### PR TITLE
Add 2019 data, close #171

### LINKED ISSUE #171: Support import of 2019 data
See #110 and https://twitter.com/cyclestreets/status/1311259150018543616.

Thanks for the heads-up @mvl22.

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Please check and review if you get a chance @agila5. Context: https://twitter.com/cyclestreets/status/1311259150018543616

File: data-raw/misc.Rmd [hunk 1]
```diff
@@ -29,7 +29,10 @@ usethis::use_data(file_names, overwrite = TRUE)
 download.file("https://github.com/ropensci/stats19/raw/master/data/file_names.rda", "file_names_old.rda")
 load("file_names_old.rda")
 file_names_old = file_names
-usethis::use_data(file_names_old)
+usethis::use_data(file_names_old, overwrite = TRUE)
+file_names = setNames(as.list(dr), dr)
+# compare the objects
+waldo::compare(file_names_old, file_names)
```
Comment (layik) [inline data-raw/misc.Rmd:35]: I see, 👍 @Robinlovelace. Guess not knowing when specific year data is released might not allow full automation. Thanks!

**Review by layik — APPROVED** [review]

**Comment by Robinlovelace** [comment]
I see this...

```

 
--


```

**Comment by Robinlovelace** [comment]
Good news, I think that's ready to go now :tada: 

``` r
remotes::install_github("ropensci/stats19", "2019")
#> Using github PAT from envvar GITHUB_PAT
#> Skipping install of 'stats19' from a github remote, the SHA1 (f1706c8a) has not changed since last install.
#>   Use `force = TRUE` to force installation
library(stats19)
#> Data provided under OGL v3.0. Cite the source and link to:
#> www.nationalarchives.gov.uk/doc/open-government-licence/version/3/


ac19 = get_stats19(year = 2019, type = "ac")
#> Files identified: DfTRoadSafety_Accidents_2019.zip
#>    http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/DfTRoadSafety_Accidents_2019.zip
#> Data already exists in data_dir, not downloading
#> Data saved at ~/stats19-data/DfTRoadSafety_Accidents_2019/Road Safety Data - Accidents 2019.csv
#> Reading in:
#> /home/robin/stats19-data/DfTRoadSafety_Accidents_2019/Road Safety Data - Accidents 2019.csv
#> date and time columns present, creating formatted datetime column
ac19
#> # A tibble: 117,536 x 33
#>    accident_index location_eastin… location_northi… longitude latitude
#>    <chr>                     <int>            <int>     <dbl>    <dbl>
#>  1 2019010128300            528218           180407   -0.154      51.5
#>  2 2019010152270            530219           172463   -0.128      51.4
#>  3 2019010155191            530222           182543   -0.124      51.5
#>  4 2019010155192            525531           184605   -0.191      51.5
#>  5 2019010155194            524920           184004   -0.200      51.5
#>  6 2019010155195            540188           185266    0.0205     51.5
#>  7 2019010155196            532424           164886   -0.0991     51.4
#>  8 2019010155198            532773           178460   -0.0890     51.5
#>  9 2019010155206            548535           188113    0.142      51.6
#> 10 2019010155207            522267           168185   -0.244      51.4
#> # … with 117,526 more rows, and 28 more variables: police_force <chr>,
#> #   accident_severity <chr>, number_of_vehicles <int>,
#> #   number_of_casualties <int>, date <date>, day_of_week <chr>, time <chr>,
#> #   local_authority_district <chr>, local_authority_highway <chr>,
#> #   first_road_class <chr>, first_road_number <int>, road_type <chr>,
#> #   speed_limit <int>, junction_detail <chr>, junction_control <chr>,
#> #   second_road_class <chr>, second_road_number <int>,
#> #   pedestrian_crossing_human_control <chr>,
#> #   pedestrian_crossing_physical_facilities <chr>, light_conditions <chr>,
#> #   weather_conditions <chr>, road_surface_conditions <chr>,
#> #   special_conditions_at_site <chr>, carriageway_hazards <chr>,
#> #   urban_or_rural_area <chr>,
#> #   did_police_officer_attend_scene_of_accident <int>,
#> #   lsoa_of_accident_location <chr>, datetime <dttm>
ve19 = get_stats19(year = 2019, type = "ve")
#> Files identified: DfTRoadSafety_Vehicles_2019.zip
#>    http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/DfTRoadSafety_Vehicles_2019.zip
#> Data already exists in data_dir, not downloading
#> Data saved at ~/stats19-data/DfTRoadSafety_Vehicles_2019/Road Safety Data- Vehicles 2019.csv
ve19
#> # A tibble: 216,381 x 23
#>    accident_index vehicle_referen… vehicle_type towing_and_arti…
#>    <chr>                     <int> <chr>        <chr>           
#>  1 2019010128300                 1 Car          No tow/articula…
#>  2 2019010128300                 2 Car          No tow/articula…
#>  3 2019010152270                 1 Car          No tow/articula…
#>  4 2019010152270                 2 Car          No tow/articula…
#>  5 2019010155191                 1 Car          No tow/articula…
#>  6 2019010155191                 2 Pedal cycle  No tow/articula…
#>  7 2019010155192                 1 Car          No tow/articula…
#>  8 2019010155194                 1 Car          No tow/articula…
#>  9 2019010155194                 2 Car          No tow/articula…
#> 10 2019010155195                 1 Car          No tow/articula…
#> # … with 216,371 more rows, and 19 more variables: vehicle_manoeuvre <chr>,
#> #   vehicle_location_restricted_lane <int>, junction_location <chr>,
#> #   skidding_and_overturning <chr>, hit_object_in_carriageway <int>,
#> #   vehicle_leaving_carriageway <int>, hit_object_off_carriageway <int>,
#> #   first_point_of_impact <chr>, was_vehicle_left_hand_drive <chr>,
#> #   journey_purpose_of_driver <chr>, sex_of_driver <chr>, age_of_driver <int>,
#> #   age_band_of_driver <int>, engine_capacity_cc <int>, propulsion_code <chr>,
#> #   age_of_vehicle <int>, driver_imd_decile <chr>, driver_home_area_type <int>,
#> #   vehicle_imd_decile <int>
ca19 = get_stats19(year = 2019, type = "ca")
#> Files identified: DfTRoadSafety_Casualties_2019.zip
#>    http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/DfTRoadSafety_Casualties_2019.zip
#> Data already exists in data_dir, not downloading
#> Data saved at ~/stats19-data/DfTRoadSafety_Casualties_2019/Road Safety Data - Casualties 2019.csv
ca19
#> # A tibble: 153,158 x 16
#>    accident_index vehicle_referen… casualty_refere… casualty_class
#>    <chr>                     <int>            <int> <chr>         
#>  1 2019010128300                 1                1 Driver or rid…
#>  2 2019010128300                 1                2 Passenger     
#>  3 2019010128300                 1                3 Passenger     
#>  4 2019010152270                 1                1 Driver or rid…
#>  5 2019010155191                 2                1 Passenger     
#>  6 2019010155192                 1                1 Pedestrian    
#>  7 2019010155194                 1                1 Driver or rid…
#>  8 2019010155194                 1                2 Passenger     
#>  9 2019010155195                 1                1 Driver or rid…
#> 10 2019010155195                 1                2 Passenger     
#> # … with 153,148 more rows, and 12 more variables: sex_of_casualty <chr>,
#> #   age_of_casualty <int>, age_band_of_casualty <chr>, casualty_severity <chr>,
#> #   pedestrian_location <chr>, pedestrian_movement <chr>, car_passenger <chr>,
#> #   bus_or_coach_passenger <chr>, pedestrian_road_maintenance_worker <chr>,
#> #   casualty_type <chr>, casualty_home_area_type <chr>,
#> #   casualty_imd_decile <chr>
```

<sup>Created on 2020-09-30 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by agila5** [comment]
Hi! I'm a little bit late to the party (as always) but I tested it locally for 2019 data and it works 👍 🎉

**Pull request merged**